### ISSUE TITLE
CMake error while configuring icub-contrib-common

### ISSUE THREAD

**Issue by xEnVrE** [robotology/robotology-superbuild#1408]
I am trying to install the `robotology-superbuild`, as done many times in the past. 

OS: Arch Linux
CMake version: 3.26.0

I am getting this error, which is not totally clear to me, the moment the superbuild tries to configure `icub-contrib-common`:

```console
[ 51%] Performing configure step for 'ICUBcontrib'
Not searching for unused variables given on the command line.
loading initial cache file /home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/YCMTmp/ICUBcontrib-cache-Release.cmake
CMake Error at CMakeLists.txt:8 (find_package):
  Could not find a configuration file for package "YARP" that is compatible
  with requested version "3.3.2".

  The following configuration files were considered but not accepted:

    /home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfig.cmake, version: 3.8.0 (64bit)



-- Configuring incomplete, errors occurred!
make[2]: *** [CMakeFiles/ICUBcontrib.dir/build.make:93: src/ICUBcontrib/CMakeFiles/YCMStamp/ICUBcontrib-configure] Error 1
make[1]: *** [CMakeFiles/Makefile2:1048: CMakeFiles/ICUBcontrib.dir/all] Error 2
make: *** [Makefile:101: all] Error 2
```

I tried to configure the repository manually in order to get the trace and I got:

```console
[xenvre@iiticublap202]$:~/robot-code/robotology-superbuild/build/src/ICUBcontrib ((v2023.02.3))$ cmake --trace ~/robot-code/robotology-superbuild/src/ICUBcontrib/
Running with trace output on.
/home/xenvre/robot-code/robotology-superbuild/src/ICUBcontrib/CMakeLists.txt(5):  cmake_minimum_required(VERSION 3.16 )
/home/xenvre/robot-code/robotology-superbuild/src/ICUBcontrib/CMakeLists.txt(6):  project(ICUBcontrib NONE )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(1):  set(CMAKE_HOST_SYSTEM Linux-6.2.7-arch1-1 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(2):  set(CMAKE_HOST_SYSTEM_NAME Linux )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(3):  set(CMAKE_HOST_SYSTEM_VERSION 6.2.7-arch1-1 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(4):  set(CMAKE_HOST_SYSTEM_PROCESSOR x86_64 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(8):  set(CMAKE_SYSTEM Linux-6.2.7-arch1-1 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(9):  set(CMAKE_SYSTEM_NAME Linux )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(10):  set(CMAKE_SYSTEM_VERSION 6.2.7-arch1-1 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(11):  set(CMAKE_SYSTEM_PROCESSOR x86_64 )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(13):  set(CMAKE_CROSSCOMPILING FALSE )
/home/xenvre/robot-code/robotology-superbuild/build/src/ICUBcontrib/CMakeFiles/3.26.0/CMakeSystem.cmake(15):  set(CMAKE_SYSTEM_LOADED 1 )
/usr/share/cmake/Modules/CMakeSystemSpecificInitialize.cmake(19):  set(CMAKE_EFFECTIVE_SYSTEM_NAME ${CMAKE_SYSTEM_NAME} )
/usr/share/cmake/Modules/CMakeSystemSpecificInitialize.cmake(21):  include(Platform/${CMAKE_SYSTEM_NAME}-Initialize OPTIONAL )
/usr/share/cmake/Modules/CMakeSystemSpecificInitialize.cmake(23):  set(CMAKE_SYSTEM_SPECIFIC_INITIALIZE_LOADED 1 )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(14):  set(APPLE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(15):  set(UNIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(16):  set(CYGWIN )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(17):  set(MSYS )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(18):  set(WIN32 )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(19):  set(BSD )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(20):  set(LINUX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(22):  function(_cmake_record_install_prefix )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(40):  include(CMakeGenericSystem )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(4):  include(CMakeInitializeConfigs )
/usr/share/cmake/Modules/CMakeInitializeConfigs.cmake(4):  include_guard(GLOBAL )
/usr/share/cmake/Modules/CMakeInitializeConfigs.cmake(8):  function(cmake_initialize_per_config_variable _PREFIX _DOCSTRING )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(6):  set(CMAKE_SHARED_LIBRARY_C_FLAGS  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(7):  set(CMAKE_SHARED_LIBRARY_CREATE_C_FLAGS -shared )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(8):  set(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(9):  set(CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(10):  set(CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG_SEP  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(11):  set(CMAKE_INCLUDE_FLAG_C -I )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(12):  set(CMAKE_LIBRARY_PATH_FLAG -L )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(13):  set(CMAKE_LIBRARY_PATH_TERMINATOR  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(14):  set(CMAKE_LINK_LIBRARY_FLAG -l )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(16):  set(CMAKE_LINK_LIBRARY_SUFFIX  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(17):  set(CMAKE_STATIC_LIBRARY_PREFIX lib )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(18):  set(CMAKE_STATIC_LIBRARY_SUFFIX .a )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(19):  set(CMAKE_SHARED_LIBRARY_PREFIX lib )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(20):  set(CMAKE_SHARED_LIBRARY_SUFFIX .so )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(21):  set(CMAKE_EXECUTABLE_SUFFIX  )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(22):  set(CMAKE_DL_LIBS dl )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(24):  set(CMAKE_FIND_LIBRARY_PREFIXES lib )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(25):  set(CMAKE_FIND_LIBRARY_SUFFIXES .so .a )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(31):  set(CMAKE_LINK_LIBRARY_USING_DEFAULT_SUPPORTED TRUE )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(33):  if(NOT DEFINED CMAKE_AUTOGEN_ORIGIN_DEPENDS )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(34):  set(CMAKE_AUTOGEN_ORIGIN_DEPENDS ON )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(36):  if(NOT DEFINED CMAKE_AUTOMOC_COMPILER_PREDEFINES )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(37):  set(CMAKE_AUTOMOC_COMPILER_PREDEFINES ON )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(39):  if(NOT DEFINED CMAKE_AUTOMOC_PATH_PREFIX )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(40):  set(CMAKE_AUTOMOC_PATH_PREFIX OFF )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(42):  if(NOT DEFINED CMAKE_AUTOMOC_MACRO_NAMES )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(43):  set(CMAKE_AUTOMOC_MACRO_NAMES Q_OBJECT Q_GADGET Q_NAMESPACE Q_NAMESPACE_EXPORT )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(47):  set_property(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS TRUE )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(49):  set(CMAKE_SKIP_RPATH NO CACHE BOOL If set, runtime paths are not added when using shared libraries. )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(51):  set(CMAKE_SKIP_INSTALL_RPATH NO CACHE BOOL If set, runtime paths are not added when installing shared libraries, but are added when building. )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(54):  set(CMAKE_VERBOSE_MAKEFILE FALSE CACHE BOOL If this value is on, makefiles will be generated without the .SILENT directive, and all commands will be echoed to the console during the make.  This is useful for debugging only. With Visual Studio IDE projects all commands are done without /nologo. )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(56):  if(DEFINED ENV{CMAKE_COLOR_DIAGNOSTICS} AND NOT DEFINED CACHE{CMAKE_COLOR_DIAGNOSTICS} )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(60):  if(CMAKE_GENERATOR MATCHES Make )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(61):  if(NOT DEFINED CMAKE_COLOR_DIAGNOSTICS )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(62):  set(CMAKE_COLOR_MAKEFILE ON CACHE BOOL Enable/Disable color output during build. )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(64):  mark_as_advanced(CMAKE_COLOR_MAKEFILE )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(66):  if(DEFINED CMAKE_RULE_MESSAGES )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(69):  if(DEFINED CMAKE_TARGET_MESSAGES )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(74):  if(NOT DEFINED CMAKE_EXPORT_COMPILE_COMMANDS AND CMAKE_GENERATOR MATCHES Ninja|Unix Makefiles )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(88):  function(GetDefaultWindowsPrefixBase var )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(180):  if(NOT DEFINED CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(185):  if(CMAKE_HOST_UNIX )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(186):  set(CMAKE_INSTALL_PREFIX /usr/local CACHE PATH Install path prefix, prepended onto install directories. )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(198):  set(CMAKE_INSTALL_DEFAULT_COMPONENT_NAME Unspecified )
/usr/share/cmake/Modules/CMakeGenericSystem.cmake(200):  mark_as_advanced(CMAKE_SKIP_RPATH CMAKE_SKIP_INSTALL_RPATH CMAKE_VERBOSE_MAKEFILE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(43):  set(CMAKE_SYSTEM_INFO_FILE Platform/${CMAKE_SYSTEM_NAME} )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(45):  include(${CMAKE_SYSTEM_INFO_FILE} OPTIONAL RESULT_VARIABLE _INCLUDED_SYSTEM_INFO_FILE )
/usr/share/cmake/Modules/Platform/Linux.cmake(1):  set(LINUX 1 )
/usr/share/cmake/Modules/Platform/Linux.cmake(2):  set(CMAKE_DL_LIBS dl )
/usr/share/cmake/Modules/Platform/Linux.cmake(3):  set(CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG -Wl,-rpath, )
/usr/share/cmake/Modules/Platform/Linux.cmake(4):  set(CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG_SEP : )
/usr/share/cmake/Modules/Platform/Linux.cmake(5):  set(CMAKE_SHARED_LIBRARY_RPATH_ORIGIN_TOKEN \$ORIGIN )
/usr/share/cmake/Modules/Platform/Linux.cmake(6):  set(CMAKE_SHARED_LIBRARY_RPATH_LINK_C_FLAG -Wl,-rpath-link, )
/usr/share/cmake/Modules/Platform/Linux.cmake(7):  set(CMAKE_SHARED_LIBRARY_SONAME_C_FLAG -Wl,-soname, )
/usr/share/cmake/Modules/Platform/Linux.cmake(8):  set(CMAKE_EXE_EXPORTS_C_FLAG -Wl,--export-dynamic )
/usr/share/cmake/Modules/Platform/Linux.cmake(12):  set(CMAKE_PLATFORM_USES_PATH_WHEN_NO_SONAME 1 )
/usr/share/cmake/Modules/Platform/Linux.cmake(18):  foreach(type SHARED_LIBRARY SHARED_MODULE EXE )
/usr/share/cmake/Modules/Platform/Linux.cmake(19):  set(CMAKE_${type}_LINK_STATIC_C_FLAGS -Wl,-Bstatic )
/usr/share/cmake/Modules/Platform/Linux.cmake(20):  set(CMAKE_${type}_LINK_DYNAMIC_C_FLAGS -Wl,-Bdynamic )
/usr/share/cmake/Modules/Platform/Linux.cmake(19):  set(CMAKE_${type}_LINK_STATIC_C_FLAGS -Wl,-Bstatic )
/usr/share/cmake/Modules/Platform/Linux.cmake(20):  set(CMAKE_${type}_LINK_DYNAMIC_C_FLAGS -Wl,-Bdynamic )
/usr/share/cmake/Modules/Platform/Linux.cmake(19):  set(CMAKE_${type}_LINK_STATIC_C_FLAGS -Wl,-Bstatic )
/usr/share/cmake/Modules/Platform/Linux.cmake(20):  set(CMAKE_${type}_LINK_DYNAMIC_C_FLAGS -Wl,-Bdynamic )
/usr/share/cmake/Modules/Platform/Linux.cmake(26):  if(NOT DEFINED _CMAKE_LINKER_PUSHPOP_STATE_SUPPORTED )
/usr/share/cmake/Modules/Platform/Linux.cmake(38):  if(_CMAKE_LINKER_PUSHPOP_STATE_SUPPORTED )
/usr/share/cmake/Modules/Platform/Linux.cmake(42):  else()
/usr/share/cmake/Modules/Platform/Linux.cmake(43):  set(CMAKE_LINK_LIBRARY_USING_WHOLE_ARCHIVE LINKER:--whole-archive <LINK_ITEM> LINKER:--no-whole-archive )
/usr/share/cmake/Modules/Platform/Linux.cmake(47):  set(CMAKE_LINK_LIBRARY_USING_WHOLE_ARCHIVE_SUPPORTED TRUE )
/usr/share/cmake/Modules/Platform/Linux.cmake(52):  set(CMAKE_LINK_GROUP_USING_RESCAN LINKER:--start-group LINKER:--end-group )
/usr/share/cmake/Modules/Platform/Linux.cmake(53):  set(CMAKE_LINK_GROUP_USING_RESCAN_SUPPORTED TRUE )
/usr/share/cmake/Modules/Platform/Linux.cmake(64):  if(DEFINED CMAKE_INSTALL_SO_NO_EXE )
/usr/share/cmake/Modules/Platform/Linux.cmake(67):  set(CMAKE_INSTALL_SO_NO_EXE ${CMAKE_INSTALL_SO_NO_EXE} CACHE INTERNAL Install .so files without execute permission. )
/usr/share/cmake/Modules/Platform/Linux.cmake(84):  set(CMAKE_LIBRARY_ARCHITECTURE_REGEX [a-z0-9_]+(-[a-z0-9_]+)?-linux-gnu[a-z0-9_]* )
/usr/share/cmake/Modules/Platform/Linux.cmake(86):  include(Platform/UnixPaths )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(10):  if(__UNIX_PATHS_INCLUDED )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(13):  set(__UNIX_PATHS_INCLUDED 1 )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(15):  set(UNIX 1 )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(19):  get_filename_component(_CMAKE_INSTALL_DIR ${CMAKE_ROOT} PATH )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(20):  get_filename_component(_CMAKE_INSTALL_DIR ${_CMAKE_INSTALL_DIR} PATH )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(28):  list(APPEND CMAKE_SYSTEM_PREFIX_PATH /usr/local /usr / ${_CMAKE_INSTALL_DIR} )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(35):  if(NOT CMAKE_FIND_NO_INSTALL_PREFIX )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(36):  list(APPEND CMAKE_SYSTEM_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(40):  if(CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(47):  _cmake_record_install_prefix()
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(23):  set(_CMAKE_SYSTEM_PREFIX_PATH_INSTALL_PREFIX_VALUE ${CMAKE_INSTALL_PREFIX} PARENT_SCOPE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(24):  set(_CMAKE_SYSTEM_PREFIX_PATH_STAGING_PREFIX_VALUE ${CMAKE_STAGING_PREFIX} PARENT_SCOPE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(25):  set(icount 0 )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(26):  set(scount 0 )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(27):  foreach(value IN LISTS CMAKE_SYSTEM_PREFIX_PATH )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(28):  if(value STREQUAL CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(31):  if(value STREQUAL CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(28):  if(value STREQUAL CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(31):  if(value STREQUAL CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(28):  if(value STREQUAL CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(31):  if(value STREQUAL CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(28):  if(value STREQUAL CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(31):  if(value STREQUAL CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(28):  if(value STREQUAL CMAKE_INSTALL_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(29):  math(EXPR icount ${icount}+1 )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(31):  if(value STREQUAL CMAKE_STAGING_PREFIX )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(35):  set(_CMAKE_SYSTEM_PREFIX_PATH_INSTALL_PREFIX_COUNT ${icount} PARENT_SCOPE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(36):  set(_CMAKE_SYSTEM_PREFIX_PATH_STAGING_PREFIX_COUNT ${scount} PARENT_SCOPE )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(50):  list(APPEND CMAKE_SYSTEM_PREFIX_PATH /usr/X11R6 /usr/pkg /opt )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(57):  list(APPEND CMAKE_SYSTEM_INCLUDE_PATH /usr/include/X11 )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(62):  list(APPEND CMAKE_SYSTEM_LIBRARY_PATH /usr/lib/X11 )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(67):  list(APPEND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES /lib /lib32 /lib64 /usr/lib /usr/lib32 /usr/lib64 )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(71):  if(CMAKE_SYSROOT_COMPILE )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(73):  else()
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(74):  set(_cmake_sysroot_compile ${CMAKE_SYSROOT} )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(79):  set(_CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES_INIT ${CMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES} ${_cmake_sysroot_compile}/usr/include )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(83):  set(_CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES_INIT ${CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES} ${_cmake_sysroot_compile}/usr/include )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(87):  set(_CMAKE_CUDA_IMPLICIT_INCLUDE_DIRECTORIES_INIT ${CMAKE_CUDA_IMPLICIT_INCLUDE_DIRECTORIES} ${_cmake_sysroot_compile}/usr/include )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(92):  unset(_cmake_sysroot_compile )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(97):  if(CMAKE_COMPILER_SYSROOT )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(106):  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB32_PATHS TRUE )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(107):  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE )
/usr/share/cmake/Modules/Platform/UnixPaths.cmake(108):  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIBX32_PATHS TRUE )
/usr/share/cmake/Modules/Platform/Linux.cmake(90):  if(NOT CMAKE_CROSSCOMPILING )
/usr/share/cmake/Modules/Platform/Linux.cmake(91):  if(EXISTS /etc/debian_version )
/usr/share/cmake/Modules/Platform/Linux.cmake(95):  if(EXISTS /etc/arch-release )
/usr/share/cmake/Modules/Platform/Linux.cmake(96):  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS FALSE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(47):  if(NOT _INCLUDED_SYSTEM_INFO_FILE )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(61):  if(CMAKE_EXTRA_GENERATOR )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(71):  if(NOT CMAKE_MODULE_EXISTS )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(72):  set(CMAKE_SHARED_MODULE_PREFIX ${CMAKE_SHARED_LIBRARY_PREFIX} )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(73):  set(CMAKE_SHARED_MODULE_SUFFIX ${CMAKE_SHARED_LIBRARY_SUFFIX} )
/usr/share/cmake/Modules/CMakeSystemSpecificInformation.cmake(77):  set(CMAKE_SYSTEM_SPECIFIC_INFORMATION_LOADED 1 )
/home/xenvre/robot-code/robotology-superbuild/src/ICUBcontrib/CMakeLists.txt(8):  find_package(YARP 3.3.2 REQUIRED )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(12):  set(PACKAGE_VERSION 3.8.0 )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(14):  if(PACKAGE_VERSION VERSION_LESS PACKAGE_FIND_VERSION )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(16):  else()
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(18):  if(3.8.0 MATCHES ^([0-9]+)\\. )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(19):  set(CVF_VERSION_MAJOR ${CMAKE_MATCH_1} )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(20):  if(NOT CVF_VERSION_MAJOR VERSION_EQUAL 0 )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(21):  string(REGEX REPLACE ^0+  CVF_VERSION_MAJOR ${CVF_VERSION_MAJOR} )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(27):  if(PACKAGE_FIND_VERSION_RANGE )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(41):  else()
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(42):  if(PACKAGE_FIND_VERSION_MAJOR STREQUAL CVF_VERSION_MAJOR )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(43):  set(PACKAGE_VERSION_COMPATIBLE TRUE )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(48):  if(PACKAGE_FIND_VERSION STREQUAL PACKAGE_VERSION )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(56):  if(CMAKE_SIZEOF_VOID_P STREQUAL  OR 8 STREQUAL  )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(61):  if(NOT CMAKE_SIZEOF_VOID_P STREQUAL 8 )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(62):  math(EXPR installedBits 8 * 8 )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(63):  set(PACKAGE_VERSION ${PACKAGE_VERSION} (${installedBits}bit) )
/home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfigVersion.cmake(64):  set(PACKAGE_VERSION_UNSUITABLE TRUE )
CMake Error at CMakeLists.txt:8 (find_package):
  Could not find a configuration file for package "YARP" that is compatible
  with requested version "3.3.2".

  The following configuration files were considered but not accepted:

    /home/xenvre/robot-code/robotology-superbuild/build/install/lib/cmake/YARP/YARPConfig.cmake, version: 3.8.0 (64bit)



-- Configuring incomplete, errors occurred!
```

**Comment by xEnVrE** [comment]
Not sure if this the right place for opening the issue. If it is better to move in within `icub-contrib-common` let me know.

cc @traversaro

**Comment by traversaro** [comment]
Indeed: https://github.com/robotology/robotology-superbuild/issues/1366 . It is an issue in CMake 3.26.0, that was fixed in 3.26.1, see https://gitlab.kitware.com/cmake/cmake/-/issues/24608 and https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8337 . Can you update your CMake version to 3.26.1 ?

**Comment by traversaro** [comment]
If you can't, a simple temporary workaround until you can use CMake 3.26.0 is to remove NONE from https://github.com/robotology/icub-contrib-common/blob/master/CMakeLists.txt#L6 .

**Comment by xEnVrE** [comment]
Thanks, it worked as you suggested in https://github.com/robotology/robotology-superbuild/issues/1408#issuecomment-1545993140.

Should I close this?

**Comment by traversaro** [comment]
> Should I close this?

I guess yes, CMake 3.26.1 has been release some time ago and CMake 3.26.0 has been the latest release of CMake just for a few days, and no distro uses at the moment, see https://repology.org/project/cmake/versions . Just to understand, why you need to use exactly CMake 3.26.0 ?

**Comment by xEnVrE** [comment]
> Just to understand, why you need to use exactly CMake 3.26.0 ?

No specific needs, It is just I didn't have time update my setup. Indeed the available version of CMake is now `cmake-3.26.3-1`.

**Comment by xEnVrE** [comment]
Closing then.
